podcasts/PodcastListViewController.swift

```diff
@@ -220,22 +220,22 @@ class PodcastListViewController: PCViewController, UIGestureRecognizerDelegate,
 
     @objc private func createFolderTapped(_ sender: UIBarButtonItem) {
         if FeatureFlag.onboardingUpdates {
-            let viewModel = WelcomeViewModel(displayType: .plus)
-            let controller = WelcomeHostingViewController(rootView: WelcomeView(viewModel: viewModel).setupDefaultEnvironment())
-            let navigationController = OnboardingNavigationViewController(rootViewController: controller)
-            viewModel.navigationController = navigationController
-
-            self.present(navigationController, animated: true)
-
-//            let coordinator = LoginCoordinator()
-//            let view = LoginLandingView(coordinator: coordinator)
-//            let controller = LoginLandingHostingController(rootView: view.setupDefaultEnvironment(), coordinator: coordinator)
-//
+//            let viewModel = WelcomeViewModel(displayType: .plus)
```

Comment:
Given we're merging to `trunk` shall we add a TODO here? Or do you think it's not necessary?